http_service/bugbug_http/app.py

```diff
@@ -1087,6 +1087,10 @@ def patch_schedules(base_rev, patch_hash):
     job = JobInfo(schedule_tests_from_patch, base_rev, patch_hash)
     data = get_result(job)
     if data:
+        # Consume the request body for POST requests before returning the cached response
+        # to avoid the client receiving a response before finishing sending the body
+        if request.method == "POST":
+            _ = request.data
```

Comment:
We could explain that we don't need to read the POST data to be able to find the response in the cache because the hash of the data is in the URL.